src/VCR.hs

```diff
@@ -33,45 +38,54 @@ import           WebMock
 data Tape = Tape {
   tapeFile :: FilePath
 , tapeMode :: Mode
-} deriving (Eq, Show)
+, tapeRedact :: Request -> Request
+}
 
 instance IsString Tape where
-    fromString file = Tape file AnyOrder
+    fromString file = Tape file AnyOrder redactAuthorization
 
 data Mode = Sequential | AnyOrder
     deriving (Eq, Show)
 
+redactAuthorization :: Request -> Request
+redactAuthorization request@Request{..} = request { requestHeaders = map redact requestHeaders }
+  where
+    redact :: Header -> Header
+    redact header@(name, _)
+      | name == hAuthorization = (name, "********")
```

Comment:
@creichert Alternatively, instead of `********`, we could use `[REDACTED]`.  I don't have any preference. Opinions?